src/Label/Events/Event.php

```diff
@@ -5,7 +5,7 @@
 use Broadway\Serializer\SerializableInterface;
 use ValueObjects\Identity\UUID;
 
-abstract class AbstractEvent implements SerializableInterface
```

Comment:
Is inderdaad veel beter en in de volgende commit is dit terug gezet [3a644d5](https://github.com/cultuurnet/udb3-php/pull/168/commits/3a644d5eeaf35f48a673cfdc964805b4e1c9cae7)